systemverilog/SystemVerilog.g4

```diff
@@ -0,0 +1,3558 @@
+	: (net_port_header | interface_port_header)? port_identifier unpacked_dimension* ('=' constant_expression)?
+	| variable_port_header? port_identifier variable_dimension* ('=' constant_expression)?
+	| port_direction? '.' port_identifier '(' expression? ')'
+	;
+
+// A.1.4 Module items
+
+elaboration_system_task
+	: '$fatal' ('(' /*finish_number*/ (/*','*/ list_of_arguments)? ')')? ';'
+	| ('$error' | '$warning' | '$info') ('(' list_of_arguments? ')')? ';'
+	;
+
+/*finish_number
+	: '0'
+	| '1'
+	| '2'
+	;
+*/
+module_common_item
+	: module_or_generate_item_declaration
+	| interface_instantiation
+	| program_instantiation
+	| assertion_item
+	| bind_directive
+	| continuous_assign
+	| net_alias
+	| initial_construct
+	| final_construct
+	| always_construct
+	| loop_generate_construct
+	| conditional_generate_construct
+	| elaboration_system_task
+	;
+
+module_item
+	: port_declaration ';'
+	| non_port_module_item
+	;
+
+module_or_generate_item
+	: attribute_instance* (parameter_override | gate_instantiation /*| udp_instantiation*/ | module_instantiation | module_common_item)
+	;
+
+module_or_generate_item_declaration
+	: package_or_generate_item_declaration
+	| genvar_declaration
+	| clocking_declaration
+	| 'default' (('clocking' clocking_identifier) | ('disable' 'iff' expression_or_dist)) ';'
+	;
+
+non_port_module_item
+	: generate_region
+	| module_or_generate_item
+	| specify_block
+	| attribute_instance* specparam_declaration
+	| program_declaration
+	| module_declaration
+	| interface_declaration
+	| timeunits_declaration
+	;
+
+parameter_override
+	: 'defparam' list_of_defparam_assignments ';'
+	;
+
+bind_directive
+	: 'bind' ((bind_target_scope (':' bind_target_instance_list)?) | bind_target_instance) bind_instantiation ';'
+	;
+
+bind_target_scope
+	: module_identifier
+	| interface_identifier
+	;
+
+bind_target_instance
+	: hierarchical_identifier constant_bit_select
+	;
+
+bind_target_instance_list
+	: bind_target_instance (',' bind_target_instance)*
+	;
+
+bind_instantiation
+	: program_instantiation
+	| module_instantiation
+	| interface_instantiation
+	| checker_instantiation
+	;
+
+// A.1.5 Configuration source text
+
+config_declaration
+	: 'config' config_identifier ';' (local_parameter_declaration ';')* design_statement config_rule_statement* 'endconfig' (':' config_identifier)?
+	;
+
+design_statement
+	: 'design' ((library_identifier '.')? cell_identifier)* ';'
+	;
+
+config_rule_statement
+	: (default_clause | inst_clause | cell_clause) liblist_clause ';'
+	| (inst_clause | cell_clause) use_clause ';'
+	;
+
+default_clause
+	: 'default'
+	;
+
+inst_clause
+	: 'instance' inst_name
+	;
+
+inst_name
+	: topmodule_identifier ('.' instance_identifier)*
+	;
+
+cell_clause
+	: 'cell' (library_identifier '.')? cell_identifier
+	;
+
+liblist_clause
+	: 'liblist' library_identifier*
+	;
+
+use_clause
+	: 'use' (library_identifier '.')? cell_identifier (named_parameter_assignment (',' named_parameter_assignment))? (':' 'config')?
```

Comment:
Didn't you forget `?` here?

```suggestion
	: 'use' (library_identifier '.')? cell_identifier (named_parameter_assignment (',' named_parameter_assignment)?)? (':' 'config')?
```

Otherwise, parentheses are excess.